packages/terra-date-time-picker/docs/DateTimePickerEvents.md

```diff
@@ -17,9 +20,25 @@ terra-date-time-picker has several callback events that are slightly different i
  - A date that is filtered by the `filterDate` prop.
  - Not an included date in the `includeDates` prop.
  - Not within the range of the `minDate` and `maxDate` props.
+* There are three parameters in the `onChange` callback prop. The first parameter is the `event` object. The second parameter is the `new date and time` value. The third parameter is the `options` object that contains the following attributes to describe the current state of the input values at the time when the `onChange` callback is triggered.
```

Comment:
What does ``new date and time`` mean here?

Reply:
`new date and time` was actually to signify the changed date and time which triggered the onChange callback.
But I agree that it can be confusing, so i have now changed it to `date and time` which is sufficient : https://github.com/cerner/terra-framework/pull/1045/commits/1e8dc74b082dc0de2c1002be51b425e1d49f9784